proto/observer/tx.proto

```diff
@@ -16,8 +17,19 @@ service Msg {
   rpc AddBlameVote(MsgAddBlameVote) returns (MsgAddBlameVoteResponse);
   rpc UpdatePermissionFlags(MsgUpdatePermissionFlags) returns (MsgUpdatePermissionFlagsResponse);
   rpc UpdateKeygen(MsgUpdateKeygen) returns (MsgUpdateKeygenResponse);
+  rpc AddBlockHeader(MsgAddBlockHeader) returns (MsgAddBlockHeaderResponse);
 }
 
+message MsgAddBlockHeader {
+  string creator = 1;
+  int64 chain_id = 2;
+  bytes tx_hash = 3;
```

Comment:
Might be a good idea to use `list { tx_hash, block_header , height } ` Some chains might produce blocks faster than us , so we might need to add more than one header per request .

Reply:
hmm.. i don't think that's a problem. zetaclient already has some measures to make sure each block is looked at in sequence.